Modules/Quotes/Tests/Unit/QuoteTaxRateServiceTest.php

```
@@ -38,13 +38,67 @@ public function it_returns_validation_rules(): void
+                if ($key === 'legacy_calculation') {
+                    return true;
+                }
+                return null;
+            }
+        }
+        
+        // Create a tax rate
+        $taxRate = \Modules\Products\Models\TaxRate::create([
+            'tax_rate_name' => 'VAT',
+            'tax_rate_percent' => 20.00,
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Products\Models\TaxRate' in method 'it_saves_tax_rate_in_legacy_mode'. (undefined)

(StaticAccess)